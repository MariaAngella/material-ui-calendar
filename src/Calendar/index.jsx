import React from "react"
import PropTypes from "prop-types"
import { Typography } from "@material-ui/core"
import { makeStyles } from "@material-ui/core/styles"
import { StylesProvider, createGenerateClassName } from "@material-ui/styles"
import styles from "./styles"

const generateClassName = createGenerateClassName({
  productionPrefix: "mu-cal"
})

const useStyles = makeStyles(styles, {
  name: "MuiRte-Editor"
})



export const CalendarComponent = props => {
  const classes = useStyles()

  return (
    <StylesProvider generateClassName={generateClassName}>
      <div><Typography>Hello World</Typography></div>
    </StylesProvider>
  )
}

CalendarComponent.propTypes = {
  
}

CalendarComponent.defaultProps = {
  
}

export default CalendarComponent
