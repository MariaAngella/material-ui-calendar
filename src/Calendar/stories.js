import React from "react"
import { createMuiTheme } from "@material-ui/core/styles"
import { ThemeProvider } from "@material-ui/styles"
import blueGrey from "@material-ui/core/colors/blueGrey"
// Import the storybook libraries
import { storiesOf } from "@storybook/react"
// Import our component from this folder
import Calendar from "./index"

const theme = createMuiTheme({
  palette: {
    text: {
      primary: blueGrey[700]
    }
  },
  overrides: {}
})

storiesOf("Calendar").add("Basic", () => (
  <ThemeProvider theme={theme}>
    <Calendar />
  </ThemeProvider>
))
