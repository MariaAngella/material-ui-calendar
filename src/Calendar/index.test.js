import React from "react"
import { createShallow, createMount } from "@material-ui/core/test-utils"
import Calendar from "./index"
import { createMuiTheme } from "@material-ui/core/styles"
import { ThemeProvider } from "@material-ui/styles"

const theme = createMuiTheme()

const Wrapped = (props = {}) => (
  <ThemeProvider theme={theme}>
    <Calendar {...props} />
  </ThemeProvider>
)

describe("Calendar", () => {
  let shallow
  let mount

  beforeAll(() => {
    shallow = createShallow()
    mount = createMount()
  })

  afterAll(() => {
    mount.cleanUp()
  })

  it("Should Render", () => {
    const component = shallow(<Wrapped />)

    expect(component).toMatchSnapshot()
  })
})
