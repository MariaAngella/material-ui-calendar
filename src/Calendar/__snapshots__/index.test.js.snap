// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar Should Render 1`] = `ShallowWrapper {}`;
